packages/tools/browserslist-config-pie/LICENCE.md

```diff
@@ -0,0 +1,17 @@
+Apache License
+Version 2.0, January 2004
+http://www.apache.org/licenses/
+
+Copyright (c) Just Eat Takeaway
```

Comment:
I believe our brand name is `Just Eat Takeaway.com` ?

Reply:
You are correct. I copied this from another package's `LICENCE` file. We seem to have a mix both in terms of the copyright and also the file naming. Some are `LICENCE`, some are `LICENSE` and some are `LICENSE.md`.

Reply:
@webbertakken If it's alright I'll fix this and the filenames in a follow-up PR